lib/matplotlib/tests/test_figure.py

```diff
@@ -3,7 +3,7 @@
 from pathlib import Path
 import platform
 from threading import Timer
-from types import SimpleNamespace
+from types import BuiltinFunctionType, SimpleNamespace
```

Comment:
I don't see where you are using this:
```suggestion
from types import SimpleNamespace
```